packages/web/app/src/components/target/laboratory/create-operation-modal.tsx

```diff
@@ -50,6 +67,24 @@ const CreateOperationMutation = graphql(`
 
 export type CreateOperationMutationType = typeof CreateOperationMutation;
 
+const createOperationModalFormSchema = z.object({
+  name: z
+    .string({
+      required_error: 'Operation name is required',
+    })
+    .min(2, {
+      message: 'Operation name must be at least 2 characters long',
+    })
```

Comment:
I see the `yup` one had `3` so let's make it `3`
```suggestion
    .min(3, {
      message: 'Operation name must be at least 3 characters long',
    })
```